lib/src/config/deprecated_option.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:meta/meta.dart';
+
+/// Class representing deprecated config option
+@immutable
+class DeprecatedOption {
+  /// Version number until which we support obsolete option
```

Comment:
The last version number which supports the option.

Reply:
fixed